services/user.js

```diff
@@ -114,12 +114,41 @@ class User {
 
     }
 
+
+    async grantToken(params, context){
+        const userInfo = context.userInfo;
+        this.isValidOrThrow([
```

Comment:
this function does not exist in CRDCDH, this is what we use in Bento to verify the session. Please verify that the user is logged in, we can use the same check used in updateMyUser